src/components/GlobalPreferences/Notifications/NotificationsPreferences.js

```diff
@@ -0,0 +1,170 @@
+                  justify-content: left;
+                `}
+                wide
+                onClick={onUpdateEmail}
+              >
+                <IconMail />
+                Update email
+              </Button>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                  justify-content: left;
+                `}
+                wide
+                onClick={onDeleteEmail}
+              >
+                <IconTrash
+                  css={`
+                    color: ${theme.negative};
+                  `}
+                />
+                Delete your email
+              </Button>
+              <Info
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+              >
+                Learn more about our{' '}
+                <Link href="https://aragon.one/email-collection.md">
+                  Email collection policy
+                </Link>
+                .
+              </Info>
+            </Box>
+          </React.Fragment>
+        }
+      />
+    </React.Fragment>
+  )
+}
+
+function EmailNotificationBox({
+  notificationsDisabled,
+  onSwitchNotificationsStatus,
+}) {
+  const handleSwitchNotificationsStatus = useCallback(() => {
+    onSwitchNotificationsStatus(!notificationsDisabled)
+  }, [notificationsDisabled, onSwitchNotificationsStatus])
+
+  return (
+    <Box heading="Signed In With Email">
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          text-align: center;
+        `}
+      >
+        <div>
+          <img
+            src={emailNotifcationIllustration}
+            width={180}
+            height={180}
+            alt=""
+          />
+        </div>
+        <div
+          css={`
+            display: flex;
+            margin-top: ${5 * GU}px;
+            align-items: center;
+          `}
+        >
+          <Switch
+            checked={!notificationsDisabled}
+            onChange={handleSwitchNotificationsStatus}
+          />
+          <span
+            css={`
+              ${textStyle('body1')};
+              margin-left: ${GU}px;
+            `}
+          >
+            Receive email notifications for all Aragon Court events.
+          </span>
+        </div>
+        <Info
+          css={`
+            margin-top: ${2 * GU}px;
+          `}
+          mode={notificationsDisabled ? 'warning' : 'info'}
+        >
+          {notificationsDisabled
+            ? `We strongly advise you to turn on these notifications. 
+            By turning off these email alerts, we won’t notify you whenever you are drafted to arbitrate a dispute or 
+            if you have any upcoming tasks. Failure to complete your jury duties will result in monetary penalties. `
+            : `We will timely notify you whenever you are drafted to arbitrate a
+          dispute, if you have any upcoming tasks and when you can start
+          claiming your rewards.`}
```

Comment:
```suggestion
            ? `We strongly advise you to enable these notifications. 
              By disabling these email alerts, you will not be notified when you are drafted to arbitrate a dispute or 
              when your upcoming tasks are due. Failure to complete your duties as a juror will result in monetary penalties.`
            : `We will notify you for any relevant Aragon Court events, such as when you are drafted to arbitrate a
              dispute, when your upcoming tasks are due, and when you have rewards to claim.`}
```

Updated the indenting and suggested an edit for the copy that reads better for me cc @dizzypaty.